charts/vector/templates/service-headless.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or (eq .Values.role "Aggregator") (eq .Values.role "Stateless-Aggregator") -}}
+{{- if and .Values.service.enabled }}
```

Comment:
It seems like the `and` here is superfluous?

Reply:
Huh - I put the `and` in the other template too 🤔